src/shared_utilities/utils.jl

```diff
@@ -86,10 +86,10 @@ Computes the weighted direct stiffness summation and updates `Y` in place.
 In the case of a column domain, no dss operations are performed.
 """
 function dss!(Y::ClimaCore.Fields.FieldVector, p::NamedTuple, t)
-    for key in propertynames(Y)
-        property = getproperty(Y, key)
-        dss_helper!(property, axes(property), p)
-    end
+    #for key in propertynames(Y)
+    #    property = getproperty(Y, key)
+    #    dss_helper!(property, axes(property), p)
+    #end
```

Comment:
Do we want to remove this, or just comment it out?

Reply:
I *think* just comment it out, since we want to come back to lateral flow at some point...and maybe if this PR causes issues, we want to be able to turn it back on? but also this stuff is preserved in git history.